app/controllers/passwordless/sessions_controller.rb

```diff
@@ -42,16 +42,16 @@ def show
       BCrypt::Password.create(params[:token])
 
       session = find_session
+      raise ExpiredSessionError if session.expired?
+
       sign_in session.authenticatable
 
       redirect_enabled = Passwordless.redirect_back_after_sign_in
-      destination = reset_passwordless_redirect_location!(User)
+      target = reset_passwordless_redirect_location!(User)
 
-      if redirect_enabled && destination
-        redirect_to destination
-      else
-        redirect_to main_app.root_path
-      end
+      redirect_to(redirect_enabled && target ? target : main_app.root_path)
```

Comment:
I like the old way better :)

Reply:
I did too, I did that to try and satisfy rubocop saying the show method was too long... Should I ignore rubocop? or is there a better solution that I missed?

Reply:
Feel free to add ignores to rubocop checks – they're only there for guidance 👍 
The method length one is called `Metrics/MethodLength`

```
# rubocop:disable Metrics/MethodLength
```

and then `# rubocop:enable ...` after the method